openssl/src/ec.rs

```diff
@@ -729,6 +729,33 @@ impl EcKey<Private> {
         }
     }
 
+    /// Constructs an public/private key pair given a curve, a private key and a public key point.
+    pub fn from_keys(
```

Comment:
Can we call this `from_private_components` to match `Rsa`?

Reply:
I've done that.